wrapper/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func addProcessToCgroup(fn string, pid int) {
+	file, err := os.OpenFile(fn, os.O_WRONLY, 0644)
+	if err != nil {
+		log.Println("unable to open", fn, "for writing :", err)
+		os.Exit(1)
```

Comment:
For consistency (here and below):
```suggestion
		log.Fatalln("unable to open", fn, "for writing :", err)
```